vscode/src/completions/logger.ts

```diff
@@ -17,6 +18,8 @@ interface CompletionEvent {
         contextSummary?: ContextSummary
         source?: string
         id: string
+        lines?: number
```

Comment:
```suggestion
        lineCount?: number
```
i think kevin wants them to be consistent and we are currently using lineCount and charCount

Reply:
@abeatrix We have historical data for `lines` going back weeks, that's why I ask

Reply:
we updated it to[ lineCount and charCount](https://github.com/sourcegraph/cody/blob/main/vscode/src/completions/logger.ts#L119) few days ago for the Measuring Cody Doc 😰

Reply:
Oh, okay, so we don't care about historical data I suspect 🙈